src/OpenTok/Broadcast.php

```diff
@@ -114,6 +114,22 @@ public function __get($name)
                 return null;
         }
     }
+
+	public function isHls(): bool
```

Comment:
Is it a good idea to return these properties as properties of the main broadcast object?

Reply:
Methods wrapping around properties like this is pretty common in PHP